aces/idt/core/structures.py

```diff
@@ -0,0 +1,197 @@
+"""Module contains a selection of helper classes used throughout the IDT
+
+"""
+import json
+from pathlib import Path
+
+import numpy as np
+
+
+class PathEncoder(json.JSONEncoder):
+    """
+    A custom JSONEncoder subclass that knows how to encode Path objects.
+    """
+
+    def default(self, obj):
+        """
+        Convert the object to a JSON serializable object
+
+        Parameters
+        ----------
+        obj : object we want to serialize
```

Comment:
**Minor**

We prefer Numpy style docstrings:

```
        Parameters
        ----------
        obj : object
            Object we want to serialize.
```

Reply:
done will keep an eye out